Modules/Devices/src/Devices.Infrastructure.Database.SqlServer/Migrations/20240403115614_ActualDeletion.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Backbone.Modules.Devices.Infrastructure.Database.SqlServer.Migrations
+{
+    /// <inheritdoc />
+    public partial class ActualDeletion : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropForeignKey(
+                name: "FK_IdentityDeletionProcessAuditLog_IdentityDeletionProcesses_IdentityDeletionProcessId",
+                table: "IdentityDeletionProcessAuditLog");
+
+            migrationBuilder.DropForeignKey(
+                name: "FK_IdentityDeletionProcesses_Identities_IdentityAddress",
+                table: "IdentityDeletionProcesses");
+
+            migrationBuilder.AddColumn<DateTime>(
+                name: "DeletionStartedAt",
+                table: "IdentityDeletionProcesses",
+                type: "datetime2",
+                nullable: false,
```

Comment:
Seems like you forgot to recreate the migration after you made the property non-nullable.

Reply:
But the property is marked as non-nullable 🤔

Reply:
Hm, yeah, idk what' I've seen there. 😆